tasks/Splitting a deck of cards.md

```diff
@@ -0,0 +1,59 @@
+> * (4)  Most important: quantity is wanted. The greater the number of ideas, the more the likelihood of winners. Come up with as many as you can that fit the cards 
+The experimenter would read these rules to participants and then instructed them to brainstorm for 10 minutes.  The instructor would then start a tape recorder and leave the room.  The experimenter would enter the room at the 3, 6 and 9 minute mark, and if no brainstorming was happening, they would prompt the group as follows; “Keep going. Please don’t forget that it’s the total number of ideas that counts in brainstorming.”  At the 10 minute mark the experimenter would enter the room, stop the tape recorder and instruct the participants to continue the experiment, starting by writing down individual hypotheses for the third or ninth card.
+
+# Procedure
+## Steps
+1. Participants are randomly broken into groups of four (or five in the case of Laughlin and Shipply) or are selected to complete the task in the nominal(individual) setting.
+2. Participants then followed instructions in an attempt to find the rule.
+
+## Roles 
+Participants worked either individually or in a group in an attempt to identify a pattern.
+
+## Instructions
+From Laughlin and Shupe 1996:
+This is an experiment in problem solving, the two groups of four will solve the same problem in separate rooms(groups are random). The objective is to find a rule that divides an ordinary deck of playing cards into two groups, examples and non examples of that rule.  Aces are 1, twos are 2 and so on, up to jacks as 11, queens as 12 and kings as 13.  The rule may be based on any characteristic of the card, I.e. the rule may be ‘diamonds’ such that all diamonds would be an example of the rule, and all hearts, spades and clubs would be non examples.  I will start you with a card that is an example of the rule.  The first step will be for each of you to write down your individual hypothesis on your individual hypothesis sheet.  Then the four of you will decide on a group hypothesis, and the recorder (designated at random) will record the groups decision on the group decision sheet.  Then you will play any of the 52 cards you choose, if the card is also an example of the rule it will be placed to the right of known example card, if not it will be placed below the last card played.  This procedure will continue for ten trials, and you will then make your final hypothesis and group hypothesis and I will tell you whether or not your final group hypothesis is correct.
+
+Laughlin and Shipply 1983 followed a similar procedure, however they did not limit based on trials, rather they gave groups/indivudals 1 hour to complete the task, if they reached the correct solution early enough, another problem set was started.
```

Comment:
I would just add a header here before the alternative versions, maybe something like: 
Alternative Version of the Task instructions